nfs/file.go

```diff
@@ -34,10 +34,7 @@ func (f *File) Read(p []byte) (int, error) {
 	}
 
 	type ReadRes struct {
-		Follows uint32
-		Attrs   struct {
-			Attrs Fattr
-		}
+		Attr  PostOpAttr
```

Comment:
Curious.  What happens the the `Follows` word?  I don't see it defined in `PostOpAttr`.

Reply:
fdawg! great to hear from you :)

Reply:
@fdawg4l Welcome! This is sort of like SAX vs DOM parsing for XML. Use of follows determines whether the gated data is even present in the XDR stream and that broke us - seems Linux had always been supplying the data, but when NetApp didn't we had misalignment in the rest of the structure.
To answer the specific query, `Follows` has been replaced with the `xdr:"union"` annotation which is in the `xdr2` package specifically to support discriminated unions.

Reply:
OOOHH!!! ok cool.  Wow.  That must've sucked.  Good find.

Reply:
tcpdump + wireshark 4TW. Wireshark has awesome XDR/NFS protocol analytics built in.

Reply:
It was the only way i made any progress was with wireshark.  Agreed.  Invaluable tool.  And it lets you go up and down the stack; tcp, rpc, nfs.